configure.ac

```diff
@@ -134,6 +134,9 @@ AM_CONDITIONAL(HAVE_PAM, [test "x$with_pam" = "xyes"])
 
 PKG_CHECK_MODULES(DBUS, dbus-1)
 
+AC_CHECK_HEADER(acl/libacl.h,[],[AC_MSG_ERROR([Cannout find libacl headers. Please install libacl-devel])])
+AC_CHECK_HEADER(libmount/libmount.h,[],[AC_MSG_ERROR([Cannout find libmount headers. Please install libmount-devel])])
```

Comment:
Can you be more specific which condition only need it? So I can improve code. I add it because when I try to compile it, it pass configure and failed in compilation phase.